version.go

```diff
@@ -405,3 +406,20 @@ func (v *Version) UnmarshalText(b []byte) error {
 func (v *Version) MarshalText() ([]byte, error) {
 	return []byte(v.String()), nil
 }
+
+// Scan implements the sql.Scanner interface.
+func (v *Version) Scan(src interface{}) error {
+	switch src := src.(type) {
+	case string:
+		return v.UnmarshalText([]byte(src))
+	case nil:
+		return nil
+	default:
+		return fmt.Errorf("Cannot scan %T as Version", src)
```

Comment:
```suggestion
		return fmt.Errorf("cannot scan %T as Version", src)
```
https://google.github.io/styleguide/go/decisions.html#error-strings

Reply:
Yeah I know, but I was following the style used in the rest of this package 😉